nats-subscriber/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+
+ENV LFH_NATS_SUBSCRIBER_HOST=nats-server
+ENV LFH_NATS_SUBSCRIBER_PORT=4222
+ENV LFH_NATS_SUBSCRIBER_TOPIC=lfh-events
+
+CMD node nats-subscriber
```

Comment:
The `exec` form of CMD is preferred - https://docs.docker.com/engine/reference/builder/#cmd.
We could probably just define `node` as the `ENTRYPOINT` and then use `nats-subscriber` as the `CMD`
```
ENTRYPOINT ["node"]
CMD ["nats-subscriber"]
```
This is actually similar to what the nats-server image does. It defines the `nats` executable as the entrypoint and then options/arguments are passed using `CMD`